cmd/gateway/setup.go

```diff
@@ -120,3 +121,14 @@ func MustValidateArguments(flagset *flag.FlagSet, validators ...ValidatorContext
 		os.Exit(1)
 	}
 }
+
+func ValidatePodIP(podIP string) error {
+	if podIP == "" {
+		return errors.New("POD_IP environment variable must be set")
```

Comment:
since this function deals with IP address pass as a parameter, I suggest removing any reference to "POD_IP" in the error messages and let the caller that context and rename it to validateIP.